Kitodo-DataManagement/src/test/java/org/kitodo/data/elasticsearch/search/SearcherIT.java

```diff
@@ -152,23 +154,24 @@ public void shouldFindDocumentByQuery() throws Exception {
         Searcher searcher = new Searcher("testsearch");
 
         String query = "{\n\"match_all\" : {}\n}";
-        JSONObject result = searcher.findDocument(query);
-        JSONObject jsonObject = (JSONObject) result.get("_source");
+        JsonObject result = searcher.findDocument(query);
+        JsonObject jsonObject = result.getJsonObject("_source");
         Integer id = getIdFromJSONObject(result);
         assertEquals("Incorrect result - id doesn't match to given number!", 1, id.intValue());
-        String title = (String) jsonObject.get("title");
+        String title = jsonObject.getString("title");
         assertEquals("Incorrect result - title doesn't match to given plain text!", "Batch1", title);
 
         query = "{\n\"match\" : {\n\"title\" : \"Batch1\"}\n}";
         result = searcher.findDocument(query);
-        jsonObject = (JSONObject) result.get("_source");
+        jsonObject = result.getJsonObject("_source");
         id = getIdFromJSONObject(result);
         assertEquals("Incorrect result - id doesn't match to given plain text!", 1, id.intValue());
-        title = (String) jsonObject.get("title");
+        title = jsonObject.getString("title");
         assertEquals("Incorrect result - title doesn't match to given plain text!", "Batch1", title);
 
         query = "{\n\"match\" : {\n\"title\" : \"Nonexistent\"}\n}";
         result = searcher.findDocument(query);
+        System.out.println(result);
```

Comment:
Is this `System.out.println()` call really needed? Maybe you should use a logger call?